pkgs/applications/misc/rofi-rbw/default.nix

```
@@ -1,28 +1,71 @@
-{ lib, buildPythonApplication, fetchFromGitHub, configargparse, setuptools, poetry-core, rbw }:
+{ lib
+, buildPythonApplication
+, fetchFromGitHub
+, configargparse
+, setuptools
+, poetry-core
+, x11Support ? false
+, waylandSupport ? false
+, wl-clipboard ? null
+, wtype ? null
+, xclip ? null
+, xdotool ? null
+, rbw }:
+
+assert waylandSupport -> wl-clipboard != null;
+assert waylandSupport -> wtype != null;
+assert x11Support -> xclip != null;
+assert x11Support -> xdotool != null;
```

I think it's unnecessary to have `null` default values. Also grouping values with their option seems to be the pattern.

```suggestion
, rbw

, waylandSupport ? false
, wl-clipboard
, wtype

, x11Support ? false
, xclip
, xdotool
}:
```